src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,26 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String brand;
+    private int mileage;
+    private int yearMade;
+    private boolean isInsured;
+    private Engine engine;
 
+    public Car(String brand,
```

Comment:
let's inline the code:
```suggestion
    public Car(String brand, int mileage, int yearMade,
```